lib/improver/nowcasting/utilities.py

```diff
@@ -142,6 +143,11 @@ def _select_orographic_enhancement_cube(precip_cube, oe_cubes):
                 Cube containing the input precipitation fields.
             oe_cubes (iris.cube.Cube or iris.cube.CubeList):
                 Cube or CubeList containing the orographic enhancement fields.
+            allowed_time_diff (int):
+                An int in seconds to define a limit to the maximum difference
+                between the datetime provided and the time points available
+                within the cube. If this limit is exceeded, then
+                an error is raised.
```

Comment:
```An int in seconds to define a limit to the maximum difference between the datetime of the precipitation cube and the time points available within the orographic enhancement cube. If this limit is exceeded, then an error is raised.```

Or something like that? There is no explicit datetime provided in this function.

Reply:
Done